src/main.rs

```diff
@@ -1,3 +1,20 @@
+//min function
+pub fn min<T: Ord>(a: T, b: T) -> T {
+    if a < b {
+        a
+    } else {
+        b
+    }
+}
+//max function
+pub fn max<T: Ord>(a: T, b: T) -> T {
+    if a > b {
```

Comment:
nit: use rust's match feature??
otherwise I'd say let's keep it clean and on one line
```rust
if a < b { a } else { b }
```